std_msgs/Header header
uint8 wall_id
geometry_msgs/Point corner_position
float64 angle
geometry_msgs/Point centroid
geometry_msgs/Vector3 normal
geometry_msgs/Point min
geometry_msgs/Point max